CHANGELOG.md

```diff
@@ -6,6 +6,17 @@
 
 ## 2.25.0
 
+### New Features
+- **AWSIoT**
+  - AWSIoT now supports retained messages ([PR #3681](https://github.com/aws-amplify/aws-sdk-ios/pull/3681))
+
+### Breaking Changes
+  - IoT: updates nullability status for ackCallback for publishing messages
```

Comment:
What is this change? Is there a PR related to this?

Reply:
I've fixed up the changelog for the next release.